crates/loro-internal/src/container/richtext/tracker.rs

```diff
@@ -364,6 +367,33 @@ impl Tracker {
         }
     }
 
+    pub(crate) fn get_target_id_latest_index_at_new_version(&self, id: ID) -> Option<Cursor> {
+        // TODO: PERF this can be sped up from O(n) to O(log(n)) but I'm not sure if it's worth it
+        let mut index = 0;
+        for span in self.rope.tree.iter() {
+            tracing::trace!("Span {:?}", span);
+            let is_activated = span.is_activated_in_diff();
+            let span_id = span.real_id();
+            let id_span = span_id.to_span(span.rle_len());
+            if id_span.contains(id) {
+                if is_activated {
+                    index += (id.counter - id_span.counter.start) as usize;
+                }
+
+                return Some(Cursor {
+                    pos: index,
+                    side: crate::stable_pos::Side::Left,
```

Comment:
How to confirm which `side` to use?

Reply:
This one is always left because it's guaranteed that the position we want (deleted, not activated) is at the left side of `index`

Reply:
Fixed with a better impl